openvpn-install.sh

```diff
@@ -245,7 +244,14 @@ LimitNPROC=infinity" > /etc/systemd/system/openvpn-server@server.service.d/disab
 	./easyrsa --batch init-pki
 	./easyrsa --batch build-ca nopass
 	./easyrsa --batch --days=3650 build-server-full server nopass
-	./easyrsa --batch --days=3650 build-client-full "$client" nopass
+	echo ""
+	read -e -p "Require password for client certificate [Y/N]: " -i "N" PASSW
```

Comment:
I would remove the default "N" on the prompt.

So i suggest this:
```suggestion
	read -e -p "Require password for client certificate [y/N], default is [N]: " PASSW
```